src/plugins/intel_gna/src/transformations/convert_dwsc_to_scaleshifts.cpp

```diff
@@ -15,8 +15,10 @@
 
 #include "utils/transformation_helper.hpp"
 
-using namespace ov::intel_gna::pass;
-using namespace ov::intel_gna::pass::helper;
+namespace ov {
+namespace intel_gna {
+namespace pass {
+using namespace helper;
 
```

Comment:
Shouldn't we disable ConvertDWSCToScaleShifts transformation for for targets >= 3.6

Reply:
No.